docs/4.operation/2.deep-log.md

```diff
@@ -0,0 +1,16 @@
+## Deep Log Inspection
+
+Deep Log Inspection, if you completed the registration of your logs in
+the FIWARE Lab service, allows you to analyse the logs. The service is
+not yet in production, but it will be available soon at:
+<https://kibana.deeplogmanager.lab.fiware.org>.
```

Comment:
It should be something like:

`[*https://kibana.deeplogmanager.lab.fiware.org*](https://kibana.deeplogmanager.lab.fiware.org)`

Nevertheless the site apparently it is not working at the moment...